source/server/listener_impl.cc

```diff
@@ -536,5 +636,68 @@ void ListenerImpl::setSocketFactory(const Network::ListenSocketFactorySharedPtr&
   socket_factory_ = socket_factory;
 }
 
+bool ListenerImpl::supportUpdateFilterChain(const envoy::config::listener::v3::Listener& config,
+                                            bool worker_started) {
+  // It's adding very little value to support filter chain only update at start phase.
```

Comment:
Is there any technical reason not to do this? Would it "just work?" If so, let's make it consistent. If not, please explain why not.